frontend/hub/collections/CollectionDetails.tsx

```diff
@@ -368,16 +369,24 @@ function CollectionDocumentationTab(props: { collection?: CollectionVersionSearc
 
   const [content, setContent] = useState<IContents>();
 
-  const { data } = useGet<CollectionDocs>(
-    hubAPI`/_ui/v1/repo/published/${collection?.collection_version?.name || ''}/${
-      collection?.collection_version?.name || ''
-    }/?include_related=my_permissions`
+  const { data } = useGet<CollectionVersionListResponse>(
```

Comment:
We could add error, and isLoading here.

Reply:
You can look here https://issues.redhat.com/browse/AAP-15074. This is the way we want to handle errors using useGet.